Source/KerbalConstructionTime/Utilities/Utilities.cs

```diff
@@ -894,11 +894,13 @@ public static BuildListVessel AddVesselToBuildList(BuildListVessel blv)
 
                     var unlockableParts = devParts.Keys.Where(p => ResearchAndDevelopment.GetTechnologyState(p.TechRequired) == RDTech.State.Available).ToList();
                     int n = unlockableParts.Count();
+                    int unlockCost = FindUnlockCost(unlockableParts);
+                    string mode = KCTGameStates.EditorShipEditingMode ? "Edit" : "Build";
                     if (unlockableParts.Any())
                     {
                         buttons = new DialogGUIButton[] {
                             new DialogGUIButton("Acknowledged", () => { }),
-                            new DialogGUIButton($"Unlock {n} part{(n > 1? "s":"")} for {FindUnlockCost(unlockableParts)} Funds and Build", () => { UnlockExperimentalParts(unlockableParts); AddVesselToBuildList(blv); })
+                            new DialogGUIButton($"Unlock {n} part{(n > 1? "s":"")} for {unlockCost} Fund{(unlockCost > 1? "s":"")} and {mode}", () => { UnlockExperimentalParts(unlockableParts); if (!KCTGameStates.EditorShipEditingMode) AddVesselToBuildList(blv); else EditShip(KCTGameStates.EditedVessel); })
```

Comment:
That delegate action is getting way too long. Should split it to multiple lines.